src/app/types/UserType.tsx

```diff
@@ -0,0 +1,20 @@
+export interface User {
+    isAdmin: boolean;
+    firstName: string;
+    lastName: string;
+    middleName?: string;
+    address: string;
+    school: string;
+    birthdate: string;
+    email: string;
+    phone: number;
+    guardian?: { 
+        guardianFirstName: string,
```

Comment:
these fields can just have the name firstName, lastName, address, etc. instead of guardianFirstName, guardianLastName, guardianAddress, etc. since they're already in the guardian object